orchestrator/gorc/Cargo.toml

```
@@ -28,7 +29,7 @@ signatory = "0.23.0-pre"
 rand_core = { version = "0.6", features = ["std"] }
 openssl-probe = "0.1.4"
 
-abscissa_tokio = { version = "0.6.0-pre.2", features = ["actix"] }
+abscissa_tokio = { version = "0.6.0-beta.1", features = ["actix"] }
```

I think the problem here is SemVer matching. Rust's implementation is unusually loose around prereleases. And since `beta` < `pre` (in terms of `Ord`), this is still pulling in `0.6.0-pre.2`, so you need to do:

```suggestion
abscissa_tokio = { version = "=0.6.0-beta.1", features = ["actix"] }
```

...to ensure it grabs a specific version (and also below)

---

In the future I'll use `alpha`/`beta` to ensure they're ordered